xla/service/gpu/priority_fusion.cc

```diff
@@ -247,13 +250,37 @@ class GpuPriorityFusionQueue {
     return !current_consumers_.empty();
   }
 
+  void updatePerformanceModelCache(HloInstruction* producer) {
```

Comment:
Rename to `UpdatePerformanceModelCache`. Google C++ code style requires that function names start from a capital letter.